<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="locale">
    <locale language="Finnish" country="Finland"/>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="rowWrapPolicy">
      <enum>QFormLayout::DontWrapRows</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <property name="verticalSpacing">
      <number>16</number>
     </property>
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Gender</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="gender">
       <property name="text">
        <string>Female</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Weight</string>
       </property>
       <property name="buddy">
        <cstring>weight</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Height</string>
       </property>
       <property name="buddy">
        <cstring>height</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="hl1">
       <item>
        <widget class="QSpinBox" name="weight">
         <property name="suffix">
          <string> kg</string>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>300</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="weight2">
         <property name="maximum">
          <number>300</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="hl2">
       <item>
        <widget class="QSpinBox" name="height">
         <property name="suffix">
          <string> cm</string>
         </property>
         <property name="maximum">
          <number>300</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="height2">
         <property name="maximum">
          <number>300</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Body mass index</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="bmi">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Interpretation</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="interp">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>309</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>Action</string>
    </property>
    <addaction name="actionReset"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuGender">
    <property name="title">
     <string>Gender</string>
    </property>
    <addaction name="actionFemale"/>
    <addaction name="actionMale"/>
   </widget>
   <addaction name="menuAction"/>
   <addaction name="menuGender"/>
  </widget>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionFemale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Female</string>
   </property>
  </action>
  <action name="actionMale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Male</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>weight</tabstop>
  <tabstop>weight2</tabstop>
  <tabstop>height</tabstop>
  <tabstop>height2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>height</sender>
   <signal>valueChanged(int)</signal>
   <receiver>height2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>weight</sender>
   <signal>valueChanged(int)</signal>
   <receiver>weight2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>height2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>height</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>weight2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>weight</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
